components/Navigation.js

```diff
@@ -74,8 +90,8 @@ const Navigation = () => {
         <FeaturedEvent event={ featuredEvents.first() } />
       }
       <nav className="h-20 fixed z-20 top-0 left-0 right-0 bg-background drop-shadow-sm md:bg-transparent md:relative md:drop-shadow-none">
-        <div className="main-content flex flex-row justify-between items-center">
-          <h3 className="logo">
+        <div className="main-content flex justify-between items-center bg-background border-b border-b-neutral-700">
+          {/* <h3 className="logo">
```

Comment:
Please us this as the logo in place of the "ReFi Spring" placeholder text from the design.